arrow-site/docs/docs/core/README.md

```diff
@@ -54,6 +54,14 @@ boilerplate and enable direct syntax including [monad comprehensions and computa
 
 </div>
 
+<div class="quickstart-coroutines-item" markdown="1">
+#### Continuations and programs
+ - [Effect]({{ '/apidocs/arrow-core/arrow.core.continuations/-effect/' | relative_url }})
+ - [EffectScope]({{ '/apidocs/arrow-core/arrow.core.continuations/-effect-scope/' | relative_url }})
+ - [EagerEffect]({{ '/apidocs/arrow-core/arrow.core.continuations/-eager-effect/' | relative_url }})
+ - [EagerEffectScope]({{ '/apidocs/arrow-core/arrow.core.continuations/-eager-effect-scope/' | relative_url }})
+</div>
+
```

Comment:
Do we have any idea how this impacts the website when we add a 5th div?

I'm also fine if we re-org _Quick Start_ & _Extensions and data types_ so that we can keep the same layout with 4 divs on the page. Adding a 5th div might impact the design/layout from the website a lot.

cc\\ @israelperezglez @franciscodr @raulraja

Reply:
We could have it under the same div as `Extension and data`, if that makes sense ?

Reply:
@i-walker can you provide a rendered image of what these changes look like before merging? ideally, we could have a build action that when selectively triggered for a commit hash temporarily deployed the site until the PR is merged.